### ISSUE TITLE
Expo.Facebook.logInWithReadPermissionsAsync redirects to wrong app if one Facebook app is used in multiple expo standalone apps (IOS + behavior: 'native')

### ISSUE THREAD

**Issue by waseemtsd** [expo/expo#4281]
### Environment

Expo CLI 2.7.1 environment info:
    System:
      OS: Windows 10
    Binaries:
      Yarn: 1.15.2 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  3.4.0.0 AI-183.5429.30.34.5452501

*Only IOS Standalone apps have this issue. On the other hand, alternate Android versions of these apps are working properly.

### Steps to Reproduce

1. Create a new facebook app, I followed complete steps from https://docs.expo.io/versions/latest/sdk/facebook/

2. Create 2 different IOS apps with different bundle Identifiers and upload to testflight to test.

### Expected Behavior

User clicks login User shall be redirected to the app which he used to login.

### Actual Behavior

User is redirected to the other app 

### Reproducible Demo

if you want to try with my apps you can download from IOS AppStore, try searching 
1. Yogofi ( https://itunes.apple.com/sg/app/yogofi/id1464406844?mt=8 )
2. Tsd Manager ( https://itunes.apple.com/sg/app/tsd-manager/id1462057527?mt=8 )
3. FollowJane Travel ( https://itunes.apple.com/sg/app/followjane-travel/id1434523951?mt=8 )

*if you Install all or any two of these three apps, try to login with these apps using Facebook and you will be redirected to one particular app every time.

If you want to try on your own
*Must use same facebook app id for both IOS app versions

here is the function i am using

async function logIn() {
  try {
    const {
      type,
      token,
      expires,
      permissions,
      declinedPermissions,
    } = await Facebook.logInWithReadPermissionsAsync('<APP_ID>', {
     permissions: ['email'],
      behavior: 'native',
    });
    if (type === 'success') {
      // Get the user's name using Facebook's Graph API
      const response = await fetch(`https://graph.facebook.com/me?access_token=${token}`);
      Alert.alert('Logged in!', `Hi ${(await response.json()).name}!`);
    } else {
      // type === 'cancel'
    }
  } catch ({ message }) {
    alert(`Facebook Login Error: ${message}`);
  }
}

**Comment by waseemtsd** [comment]
Any updates on this?

**Comment by saiyam-sage** [comment]
I am also facing this problem in the case of multiple apps. Any idea about this problem?

**Comment by waseemtsd** [comment]
@saiyam-sage I have created two different apps on facebook for now + a workaround for firebase-facebook login. Because firebase doesn't allow two facebook apps for one firebase project. If you are using firebase  to log in then you can do the following:
1. Use 2 different facebook apps
****************************now for 1st app*************************************
/////This function will use firebase-facebook authprovider to get user data & login with firebase/////
authenticate = (token) => {
    const provider = firebase.auth.FacebookAuthProvider;
    const credential = provider.credential(token);
    return firebase.auth().signInAndRetrieveDataWithCredential(credential);
  }
//////////////////////////////////authenticate Function end////////////////////////////////////////////////

const { type, token } = await Expo.Facebook.logInWithReadPermissionsAsync(ADD_ID1, { permissions:['email'], behavior: 'native' }) //this will get permissions to access user data
if (type === 'success') {
var userData= await this.authenticate(token); //// you'll get user's data here using firebase
}
**********************************************************************************

********************************now for 2nd app**********************************
const { type, token } = await Expo.Facebook.logInWithReadPermissionsAsync(ADD_ID2, { permissions:['email'], behavior: 'native' })
      if (type === 'success') {
             const response = await fetch(`https://graph.facebook.com/me?fields=id,name,email&access_token=${token}`); //// you'll get user's data here using graph api instead of firebase
            var userDataFb = (await response.json());

/////////////////////this is the workaround for 2nd app//////////////////////////////////////
/////I am creating user with email and a hardCoded password///////////

firebase.auth().createUserWithEmailAndPassword(userDataFb.email, 'setDefaultPasswordHere')
            .then(async function (data) { console.log(data) }.bind(this))
            .catch(e => console.log(e))
///////////////////////////////////////////////////////////////////////////////////////////

******************************************************************************************


Basically, this is what i am doing
1. For the first app I am using standard firebase-facebook auth provider etc.
2. For 2nd app I am using graph api to get user's data and manually creating user account on firebase without using firebase-facebook auth provider etc.


Warning: This is not a recomended way of resolving this issue but I haven't found a proper solution for this problem, neither was helped by expo community. But, This workaround has solved my issue for now.

let me know if you still have an issue... cheers

**Comment by sjchmiela** [comment]
To be able to use one Facebook App ID in multiple mobile apps and still make it redirect to proper one, one will need to use `FacebookUrlSchemeSuffix` as per [Facebook FAQ](https://developers.facebook.com/docs/ios/troubleshooting#faq_112617202494710).

Just set a custom `FacebookUrlSchemeSuffix` `infoPlist` property (in `app.json`) and add the same suffix to `facebookScheme` property in `app.json` before building an iOS app.

**Comment by musemind** [comment]
@sjchmiela @cruzach  just for clarification. I made these changes:

1. Set "URL Scheme Suffix" in the Facebook iOS settings
![image](https://user-images.githubusercontent.com/11320794/73258163-6b226380-41c5-11ea-8b07-3496161096a7.png)

2. Set `expo.facebookScheme` to `fb570xxxxx917foodly` in app.json

3. Set `expo.ios.infoPlist.FacebookUrlSchemeSuffix` to `foodly` in app.json

Does this look correct to you?
Trying to fix https://github.com/expo/expo/issues/6112
